anyway/backend_constants.py

```diff
@@ -55,10 +55,18 @@ class AccidentType:
         COLLISION_WITH_AN_ANIMAL = 19
         DAMAGE_CAUSED_BY_A_FALLING_LOAD_OFF_A_VEHICLE = 20
 
+    class DriverType(object):
```

Comment:
In Python 3, apart from compatibility between Python 2 and 3, there is no need to inherit from object.
We only use Python 3.
Read more about it in :
https://stackoverflow.com/questions/4015417/python-class-inherits-object